client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+    <div>
+        <h4>{{ l("Edit Collection Attributes") }}</h4>
+        <b-alert show variant="warning" dismissible>
+            {{
+                l(
+                    "Collections are immutable. This means there will be some things you cannot change without creating a new collection. "
+                )
+            }}
+        </b-alert>
+        <div v-if="jobError">
+            <b-alert show variant="danger" dismissible>
+                {{ errorMessage }}
+            </b-alert>
+        </div>
+        <b-tabs content-class="mt-3">
+            <b-tab>
+                <template v-slot:title> <i class="fa fa-bars"></i>{{ l(" Attributes") }}</template>
+                <b>{{ l("Name: ") }}</b> <i>{{ collectionName }}</i>
+                <br />
+                <b>{{ l("Collection Type: ") }}</b> <i>{{ collectionType }}</i>
+                <br />
+                <b>{{ l("Elements: ") }}</b> <br />
+                <div v-for="element in collectionElements" :key="element">{{ element.element_identifier }} <br /></div>
+            </b-tab>
+            <b-tab>
+                <template v-slot:title> <i class="fa fa-table"></i>{{ l(" Database/Build") }}</template>
```

Comment:
Is there a reason for the leading / trailing whitespace in many of the localization calls in this PR ?

Reply:
There needs to be a small space between the icon and the text, or else they sort of clobber each other. I can put another character in there that's not in the string to hold the spot, but something has to go there for it to look nice.

Reply:
I changed them to have &nbsp between the icon and the text - that should make things look a little better, I hope.